Applications/DataExplorer/VtkVis/VtkStationSource.h

```diff
@@ -36,16 +34,16 @@ class VtkStationSource : public vtkPolyDataAlgorithm, public VtkAlgorithmPropert
 
     /// Returns the colour lookup table generated for boreholes.
     /// This method should only be called after the colour lookup table has actually been build (via RequestData() or setColorLookupTable()).
-    const std::map<std::string, GeoLib::Color>& getColorLookupTable() const
+    const std::map<std::string, DataHolderLib::Color>& getColorLookupTable() const
         { return _colorLookupTable; }
 
     /// Returns the type of observation site represented in this source object
     GeoLib::Station::StationType getType() const
         { return static_cast<GeoLib::Station*>((*_stations)[0])->type(); };
 
     /// Sets a predefined color lookup table for the colouring of borehole stratigraphies
-    int setColorLookupTable(const std::string &filename)
-        { return GeoLib::readColorLookupTable(_colorLookupTable, filename); }
+//    int setColorLookupTable(const std::string &filename)
+//        { return GeoLib::readColorLookupTable(_colorLookupTable, filename); }
```

Comment:
I'd rather keep this for now as this is another part of the DE where lookup-tables are used but it would need a complete rewrite. I don't want to make this PR larger than it already is.